### DISCUSSION TITLE
PHP and Foreign Function Interface

### DISCUSSION THREAD

**Discussion by boan-jfm** [libvips/libvips#3649] [General]
I do not think this is correct anymore;

>**Dynamic language without FFI**
>PHP does not have a useful FFI..
> -- https://github.com/libvips/libvips/blob/v8.14.4/doc/binding.md?plain=1#L172

php-vips uses FFI: https://github.com/libvips/php-vips/blob/ce833a6d38cea395aad5cd24f9c142ed36ab70b4/src/FFI.php#L220

FFI is supported since PHP 7.4: https://wiki.php.net/rfc/ffi

**Comment by jcupitt** [comment]
You're right, that needs updating. Thanks!

https://github.com/libvips/libvips/commit/888e7f07cf7bc0e8cf6afada61c4c1069cb3a0eb
